scipy/stats/stats.py

```diff
@@ -2516,6 +2517,116 @@ def iqr(x, axis=None, rng=(25, 75), scale='raw', nan_policy='propagate',
     return out
 
 
+
+def mad(x, axis=None, nan_policy='propagate', keepdims=False):
+    """
+    Compute the median absolute deviation (MAD) of the data along the
+    specified axis [1]_.
```

Comment:
Could you make this fit on 1 line in <80 characters, and move the citation to the next paragraph? Reason: this one line is extracted into the summary table of functions listed on the `stats` doc page.